src/components/Menu/MenuItem.tsx

```diff
@@ -193,6 +220,10 @@ class MenuItem extends AutoControlledComponent<ReactProps<MenuItemProps>, MenuIt
               defaultProps: { xSpacing: !!content ? 'after' : 'none' },
             })}
           {content}
+          {showSubmenuIndicatorIcon &&
```

Comment:
Do we need `hideSubmenuIndicator` at all?

```jsx
<Menu submenuIndicatorVertical={null} submenuIndicatorVertical={null} />
```

Reply:
Nope, removed.